runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsOverlay.java

```diff
@@ -127,9 +189,51 @@ else if (plugin.getRoomBlacklist().contains(room.getBoss().getName().toLowerCase
 						color = Color.RED;
 					}
 
+					String bossName = room.getBoss().getName();
+					if (config.showRecommendedItems())
+					{
+						switch (RaidRoom.Boss.fromString(bossName))
+						{
+							case GUARDIANS:
+								//guardians = true;
+								itemIds.add(11920);
+								break;
+							case MUTTADILES:
+								//muttadiles = true;
```

Comment:
What's with all of the comments?

Reply:
I filled out all of the room cases, then commented out the booleans of the ones that were not being used used.

Reply:
if they aren't being used, instead of commenting them out you should just remove them

Reply:
I have removed the lines in question as requested.